packages/firestore/src/remote/remote_store.ts

```diff
@@ -136,82 +152,66 @@ export class RemoteStore implements TargetMetadataProvider {
    * Starts up the remote store, creating streams, restoring state from
    * LocalStore, etc.
    */
-  start(): Promise<void> {
-    return this.enableNetwork();
-  }
+  async start(): Promise<void> {
+    // Load any saved stream token from persistent storage
+    this.writeStream.lastStreamToken = await this.localStore.getLastStreamToken();
```

Comment:
We are doing this twice now. Once in`start()` and then right below in `enableNetwork()`.

Reply:
Oops, thanks! I meant to remove this one.